### PR TITLE
Link added

### PR DESCRIPTION
**Pull request by ameissne**
Done

---
This change is [Reviewable](https://reviewable.io/reviews/canoo/dolphin-platform/506)

### CODE REVIEW

**Comment by zimmi** [comment]
Good job! :)

File: documentation/src/docs/asciidoc/validation.adoc [hunk 1]
```diff
@@ -1,7 +1,6 @@
 
 = Validation Support
-
-The Dolphin Platform provides the optional `dolphin-platform-bean-validation` module that adds Java Bean Validation
+The Dolphin Platform provides the optional `dolphin-platform-bean-validation` module that adds http://beanvalidation.org/[Java Bean Validation]
```
Comment (zimmi) [inline documentation/src/docs/asciidoc/validation.adoc]: Nice!

**Comment by hendrikebbers** [comment]
@ameissne please add or create issue :)

**Comment by ameissne** [comment]
Issue; https://github.com/canoo/dolphin-platform/issues/504

**Comment by hendrikebbers** [comment]
lgtm

**Pull request merged**